PyGitUp/gitup.py

```diff
@@ -580,12 +580,13 @@ def print_error(self, error):
 
 
 @click.command(epilog=EPILOG)
-@click.option('--version', is_flag=True,
+@click.option('--version', '-v', is_flag=True,

```

Comment:
In many cases `-v` means verbose and `-V` means version (for example with the Python interpreter), so this may confuse some people. What do you think?

Reply:
Agreed, that's probably better :)

I suppose I was on autopilot a bit, I'll change it

Reply:
It's done :)